routes/marketdata.js

```diff
@@ -1,54 +1,42 @@
 const express = require('express');
-const firebase = require('firebase');
-const config = require('./config.js');
 const router = express.Router();
-
-if (!firebase.apps.length) {
-    firebase.initializeApp(config.config);
-}
-// Creates connection to database.
-const db = firebase.database();
-// Links to head of database.
-const ref = db.ref("live_weller");
-// Links to markets list.
-const marketsRef = ref.child("markets");
+const db = require('../db.js');
 
 router.get('/:marketKey', function(req, res, next) {
-    const market = marketsRef.child(req.params.marketKey);
-
-    market.once('value', function(snapshot) {
-        const childData = snapshot.val();
+    db.exportAllMarkets().then(function(result) {
+        const markets = result.val();
+        const market = markets[req.params.marketKey];
```

Comment:
instead of pulling all the markets from the database and filtering for a specific one, can we just pull that specific market document from Firebase? I remember we implemented that in an earlier PR, so we can add a function for that in the database module

Reply:
For sure